.changeset/mean-lies-invent.md

```diff
@@ -0,0 +1,5 @@
+---
+"@navikt/ds-react": minor
```

Comment:
Litt usikker på om dette er en minor eller patch for å være ærlig. Det er jo ikke en bug at man må fikse det selv, men at man nå slipper å sette tabIndex selv er ikke akkurat en **stor** feature.